site/src/docs/components/header/index.mdx

```diff
@@ -64,14 +64,14 @@ Coming soon!
 
 Coming soon!
 
-### With navigation language selector
+#### With navigation language selector
 
 - Using the Header.NavigationLanguageSelector is optional. Use the navigation language selector to define site languages. 
-- Header has onDidChangeLanguage function, where user can define a callback function which gets the selected language key (value) as parameter.
-- See Example for Header.NavigationLanguageSelector <Link openInNewTab openInNewTabAriaLabel="Opens in a new tab" href="/storybook/react/?path=/story/components-header--minimal-with-localization">in Storybook</Link>
+- Header has <code>onDidChangeLanguage => onDidChangeLanguage</code> function, where user can define a callback function which gets the selected language key (value) as parameter.
```

Comment:
Sorry, I didn't mean to include => stuff in there. Just <code>onDidChangeLanguage</code>